include/openmc/cell.h

```diff
@@ -108,6 +154,12 @@ class Cell {
   //! Get the BoundingBox for this cell.
   virtual BoundingBox bounding_box() const = 0;
 
+  //! Get a vector of surfaces in the cell
+  virtual vector<int32_t> surfaces() const = 0;
+
+  //! Check if the cell region expression is simple
+  virtual bool is_simple() const = 0;
```

Comment:
You can either 1) give this virtual function a default implementation that returns true, or 2) override it in the `DAGCell` class with a method that returns true.